### PR TITLE
Add selfupdate command

### PR DESCRIPTION
**Pull request by uberhacker**
Thank you so much for contributing code to lando!

We will get to your PR as soon as we can but in the meantime you might want to
check to make sure you have done the following. **The more boxes you can check
the easier it will be for us to merge in your changes with confidence**

- [ ] My code passes relevant CI status checks
- [ ] My code includes a test if applicable ([see here](https://docs.devwithlando.io/dev/testing.html))
- [ ] My code includes an update to the `CHANGELOG` ([see here](https://github.com/kalabox/lando/tree/master/docs/changelog))
- [x] My code includes documentation updates if relevant.
- [ ] I have pinged @pirog/@reynoldsalec/@serundeputy when I think my code is ready for prime time.

If you are unclear about any of the above please add comments below so we can
improve our process.

Tested and working on:
- [x] Linux
- [ ] Mac OS X
- [ ] Windows Pro

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
   "dependencies": {
     "bluebird": "^3.4.1",
     "cli-table": "^0.3.1",
+    "deasync": "^0.1.12",
```
Comment (dustinleblanc) [inline package.json]: I'm still learning the whole sync/async thing with node, but this *feels* like a different direction than the typical promisified direction we're taking in the rest of the code. I know that a lot of the first scripts I wrote weren't fully promisified. @pirog what do you think about that?

File: package.json [hunk 2]
```diff
@@ -63,6 +64,7 @@
     "js-yaml": "^3.4.6",
     "jsonfile": "^2.4.0",
     "kalabox-stats-client": "^0.2.1",
+    "linus": "git+https://github.com/tomas/linus.git#v0.0.5",
```
Comment (dustinleblanc) [inline package.json]: This util looks handy! It has some pretty low usage numbers though. @pirog would you want to switch to something like this and add a dependency or just keep doing it ourselves?

Comment (uberhacker) [reply package.json]: I wish they had a sync method.  That would definitely help due to the current use case.

**Comment by uberhacker** [comment]
@dustinleblanc: I appreciate the feedback.

**Comment by uberhacker** [comment]
Closing due to merge conflicts and like an idiot, I pushed up to the master branch.

**Pull request closed without merging**